src/index.html

```diff
@@ -5,10 +5,36 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo-link">
+        <img src="images/logo.png" alt="logo Moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#Apple" class="nav__list__item is-active">APPLE</a></li>
```

Comment:
I would like you to change content all of hrefs to lowercase:
"#Apple" --> "#apple"
"#Samsung" --> "#samsung"
And so on

Also, the content of `a` tag should be lowercase and you should change it with help of css, try to use `text-transform: uppercase;`

Reply:
Let's also simplify the name of this class
How about `nav__link` instead of `nav__list__item`?

Reply:
also please make start `<a>` from new line
```suggestion
      <li>
          <a href="#Apple" class="nav__list__item is-active">APPLE</a>
      </li>
```